ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+#SERVICE_PROJECT="openshift-installer-shared-vpc"
+BASE_DOMAIN="installer.gcpxpn.devcluster.openshift.com"
+
+cat > "${PATCH}" << EOF
+baseDomain: ${BASE_DOMAIN}
+featureSet: TechPreviewNoUpgrade
+credentialsMode: Passthrough
+platform:
+  gcp:
+    computeSubnet: ${HOST_PROJECT_COMPUTE_SUBNET}
+    controlPlaneSubnet: ${HOST_PROJECT_CONTROL_SUBNET}
+    createFirewallRules: Disabled
+    network: ${HOST_PROJECT_NETWORK}
+    networkProjectID: ${HOST_PROJECT}
+    publicDNSZone:
+      id: installer-xpn-service-project
+      project: ${HOST_PROJECT}
```

Comment:
Personally I would like to see this job use the defaults--so to not use the `publicDNSZone` block. Also, I do not think there is an existing zone with that ID in the host project.